app.rb

```diff
@@ -0,0 +1,20 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/tweet'
+
+class Chitter < Sinatra::Base
+    configure :development do
+        register Sinatra::Reloader
+    end
+
+    get '/' do
+        'tweet tweet'
```

Comment:
homepage at the moment is quite bare, looking at your ruby code I know I should go to /tweets but a link from the homepage would be helpful